### PR TITLE
add postgresql production variables for abid

### PR DESCRIPTION
**Pull request by kayiwa**
We are migrating abid to the new replicating postgresql environment

### CODE REVIEW

File: group_vars/postgresql/production.yml [hunk 1]
```diff
@@ -34,17 +34,15 @@ postgres_port: 5432
 postgres_admin_user: "{{ postgresadmin }}"
 
 datadog_api_key: "{{ vault_datadog_key }}"
-datadog_service_name: postgresql
-datadog_environment: prod
 datadog_config:
   log_enabled: true
-  process_enabled: true
-datadog_typed_checks:
```
Comment (carolyncole) [inline group_vars/postgresql/production.yml:42]: @kayiwa why did you remove the typed check?  This is the newer and more correct configuration for datadog

**Review by carolyncole — CHANGES_REQUESTED** [review]

File: group_vars/postgresql/production.yml (hunk 1 above)
Comment (kayiwa) [reply group_vars/postgresql/production.yml:42]: Oh, I didn't realize this @carolyncole I am still debugging this. OK for this to go in? if I make it an issue.
